### PR TITLE
Fix edit expiration

### PR DESCRIPTION
**Pull request by peolic**
- I noticed the UI did not display the correct expiration time for destructive edits.
Making it a field makes it easy to update the conditions that label an edit as destructive.
(I was considering labelling performer renames as destructive when I noticed the issue)

- `ScenesList` sort order selector was wider than needed.

### CODE REVIEW

**Comment by peolic** [comment]
On the topic of edit voting, I raised this on Discord not too long ago:
Should the user who submitted an edit be able to see the current vote score (just the score, without the buttons) on their own edit?

File: frontend/src/components/editCard/VoteBar.tsx [hunk 1]
```diff
@@ -26,13 +26,23 @@ const VoteBar: FC<Props> = ({ edit }) => {
...
-    edit.status !== VoteStatusEnum.PENDING ||
-    !canVote(auth.user) ||
-    auth.user?.id === edit.user?.id
-  )
+  if (edit.status !== VoteStatusEnum.PENDING || !canVote(auth.user))
     return <></>;
 
+  if (auth.user?.id === edit.user?.id)
+    return (
+      <div className={CLASSNAME}>
```
Comment (peolic) [inline frontend/src/components/editCard/VoteBar.tsx]: I think it looked bad without the buttons when I used the same classes here. One sec...

Comment (ghost) [reply frontend/src/components/editCard/VoteBar.tsx]: image
Looks ok on my end.

Comment (peolic) [reply frontend/src/components/editCard/VoteBar.tsx]: Before
before
After
after

**Comment by peolic** [comment]
@InfiniteTF
I thought the margin at the end was a little weird? Only saw your comment after I pushed

File: frontend/src/components/editCard/VoteBar.tsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/editCard/VoteBar.tsx]: Yup, that's better. 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**